spec/models/merchant_spec.rb

```diff
@@ -66,5 +66,10 @@
 
       expect(@meg.pending_orders.sort).to eq([order_2, order_3])
     end
+
+    # it "disable_items" do
+    #   @meg.disable_items
+    #   expect(@tire.disabled).to eq(true)
+    # end
```

Comment:
Can we take this out or saving it for later?

Reply:
I just removed this -- not necessary anymore since we aren't using model methods for item disabling.